### PR TITLE
Properly handle storage permission on 8.1+

### PR DESCRIPTION
**Pull request by nyancrimew**
Prompt the user for the Storage permission with an explaination and disable blur without it to prevent crashes.

@divadsn a quick look over this would be appreciated

Manually tested on RR O 8.1 in all ways I was able to imagine 😉

### CODE REVIEW

**Comment by divadsn** [comment]
@deletescape added to my list, I will be back next Sunday.

File: app/src/main/java/ch/deletescape/lawnchair/Launcher.java [hunk 1]
```diff
@@ -366,6 +368,22 @@ public void run() {
...
+            if(ActivityCompat.shouldShowRequestPermissionRationale(this, Manifest.permission.READ_EXTERNAL_STORAGE)) {
+                final Launcher _this = this;
+                DialogInterface.OnClickListener listener = new DialogInterface.OnClickListener() {
+                    @Override
+                    public void onClick(DialogInterface di, int i) {
+                        Utilities.requestStoragePermission(_this);
+                    }
+                };
+                new AlertDialog.Builder(this).setTitle(R.string.title_storage_permission_required)
+                        .setMessage(R.string.content_storage_permission_required).setPositiveButton(android.R.string.ok, listener)
```
Comment (msfjarvis) [inline app/src/main/java/ch/deletescape/lawnchair/Launcher.java:382]: nit: line length

**Review by msfjarvis — APPROVED** [review]
LGTM

**Review by divadsn — CHANGES_REQUESTED** [review]
Storage permissions are also required for extracting the accent color for the dock background, which is also enabled by default and causes crashes on Oreo 8.1

**Comment by nyancrimew** [comment]
@divadsn updated the PR according to your comment, was able to reproduce after clearing data

**Comment by divadsn** [comment]
Pls fix codacy @deletescape

**Comment by divadsn** [comment]
FFS STOP IT TRAVIS, YOU AREN'T SUPPOSED TO BE TRIGGERED ON PR ANYMORE.

**Comment by TheComputerGuy96** [comment]
Travis, just REEEEEEEEEEEEEEEEE

**Review by divadsn — APPROVED** [review]

**Pull request merged**